xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm

```diff
@@ -34,13 +34,23 @@
 </div>
 #if ($editor != 'inline')
 <div id="titleinput" class="form-group">
-  <label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
-  <input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>
+    <dl>
+        <dt>
+          <label for="xwikidoctitleinput">Title</label>
```

Comment:
You are not using `$services.localization.render('core.editors.content.titleField.label')`.
This is really important for translations.

Reply:
It was originally like this, so it's another issue right?
I can create it in JIRA and fix it. Thanks!

Reply:
No it wasn't originally like this. What you might see in the browser console tools is rendered through this code.
In your case you are removing the velocity code that is supposed to get the translation for the word `Title`.
For the Title label as in:
```
<label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
```
`$services.localization.render('core.editors.content.titleField.label')` part gets the translation.
You are using:
```
<label for="xwikidoctitleinput">Title</label>
```
While it should be the same. You don't need to change that.
Please see the changes in both files side by side.

Reply:
Oh, yes I see it now!